src/Controls/tests/Core.UnitTests/Layouts/FlexLayoutTests.cs

```diff
@@ -80,5 +81,53 @@ public void FlexLayoutRecognizesVisibilityChange()
 			// now that the first view is not visible
 			Assert.True(whenInvisible != whenVisible);
 		}
+
+		/*
+		 * These next two tests deal with unconstrained measure of FlexLayout. Be default, FL
+		 * wants to stretch children across each axis. But you can't stretch things across infinity
+		 * without it getting weird. So for _measurement_ purposes, we treat infinity as zero and 
+		 * just give the children their desired size in the unconstrained direction. Otherwise, FL
+		 * would just set their flex frame sizes to zero, which can either cause blanks or layout cycles,
+		 * depending on the target platform.
+		 */
+
+		(IFlexLayout, IView) SetUpUnconstrainedTest() 
+		{
+			var root = new Grid(); // FlexLayout requires a parent, at least for now
+			var flexLayout = new FlexLayout() as IFlexLayout;
+
+			var view = Substitute.For<IView>();
+			var size = new Size(100, 100);
+			view.Measure(Arg.Any<double>(), Arg.Any<double>()).Returns(size);
+
+			root.Add(flexLayout);
+			flexLayout.Add(view);
+
+			return (flexLayout, view);
+		}
+
+		[Fact]
+		public void UnconstrainedHeightChildrenHaveHeight()
+		{
+			(var flexLayout, var view) = SetUpUnconstrainedTest();
+
+			_ = flexLayout.CrossPlatformMeasure(400, double.PositiveInfinity);
```

Comment:
funny, does the compiler complain about this? if we ignore the _, the code does not care does it?

Reply:
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/discards

Reply:
would be needed in a tuple decomposition, for a single value it is not. That is what I a m saying.

Reply:
From the article, emphasis mine: 

"A discard communicates intent to the compiler *and others that read your code*: You intended to ignore the result of an expression."

I'm not putting it there for the compiler. The point is to make it clear to someone reading the test that we don't care about the return value of that method for the purposes of the test; that it's not a mistake or an oversight. The author recognizes that the method returns a value, but the value is not necessary for the task at hand. 

Also, from the article:

"Discards make the intent of your code clear. A discard indicates that our code never uses the variable. They enhance its readability and maintainability."

"You can use a standalone discard to indicate any variable that you choose to ignore."

Reply:
Unfortunately, the emphasis is wrong since no were in the article states that a single return value should be explicitly discarded. But rather than focus on my interpretation of the article, there is a SPECIFIC code analysis error for this exact usage of the discards, it is [IDE0110](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0110). 

> Remove unnecessary discard (IDE0110)

So unless there is a setting in the project that will disable the warning from the code analysis, or if we are using the code analysis (I doubt), you will have to add the following:
```suggestion
#pragma warning disable IDE0110
			_ = flexLayout.CrossPlatformMeasure(400, double.PositiveInfinity);
#pragma warning restore IDE0110
```

As I said, the documentation of the unnecessary code rules is a good one to avoid writing code that is not needed. https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/unnecessary-code-rules 

From the docs:

* Readability: Code that is unnecessarily degrading readability. For example, [IDE0001](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0001) flags unnecessary type-name qualifications.
* Maintainability: Code that is no longer used after refactoring and is unnecessarily being maintained. For example, [IDE0051](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0051) flags unused private fields, properties, events, and methods.
* Performance: Unnecessary computation that has no side effects and leads to unnecessary performance overhead. For example, [IDE0059](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0059) flags unused value assignments where the expression to compute a value has no side effects.
* Functionality: Functional issue in code leading to required code being rendered redundant. For example, [IDE0060](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0060) flags unused parameters where the method accidentally ignores an input parameter.


Feel free to land as is, but bare in mind that you will have to disable a code analysis rule if they are ever used in the future and the there are several parts of the documentation that are against this pattern.

Reply:
IDE0110 is for unnecessary discard _patterns_. It won't be triggered by the code in this test. However, _not_ using the discard _will_ trigger [IDE0058](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0058).  You can try it for yourself - drop this into a project and crank the code style rules up to 11:

```
internal class Program
{
    static int counter;

    private static void Main(string[] args)
    {
        NotIDE0010_Violation();
        IDE0010_Violation("test");
        IDE0058_Violation();

        int ThisHasSideEffects()
        {
            counter += 1;
            return new Random().Next();
        }

        void IDE0010_Violation(object o)
        {
            switch (o)
            {
                case int _:
                    Console.WriteLine("Value was an int");
                    break;
                case string _:
                    Console.WriteLine("Value was a string");
                    break;
            }
        }

        void NotIDE0010_Violation()
        {
            _ = ThisHasSideEffects();
        }

        void IDE0058_Violation()
        {
            ThisHasSideEffects();
        }
    }
}
```

I just tested it on 17.4.1.